src/server/handlers/thirdpartyRequests/transactions/{ID}/authorizations.ts

```diff
@@ -87,7 +84,7 @@ export async function post (
 
     if (error instanceof NotFoundError) {
       // **************************
```

Comment:
for multiline comments use standard convention, don't use so many asterisks - they are expensive!
/** 
*
*/